snuba/cli/multistorage_consumer.py

```diff
@@ -192,6 +199,27 @@ def multistorage_consumer(
         ):
             raise ValueError("storages cannot be located on different Kafka clusters")
 
+    metrics = MetricsWrapper(
+        environment.metrics,
+        "consumer",
+        tags={
+            "group": consumer_group,
+            "storage": "_".join([storage_keys[0].value, "m"]),
+        },
+    )
+    # Collect metrics from librdkafka if we set stats_collection_frequency_ms
+    if stats_collection_frequency_ms and stats_collection_frequency_ms > 0:
+
+        def stats_callback(stats_json: str) -> None:
+            stats = rapidjson.loads(stats_json)
+            metrics.gauge("total_queue_size", stats.get("replyq", 0))
```

Comment:
while we're here is there anything else we want to send?

Reply:
I'm going to hold off for now until I have a better understand of what each of the metrics are actually doing.